behavior_tests/src/change-filename-extension/main.cu

```diff
@@ -0,0 +1,36 @@
+// UNSUPPORTED: -windows-
+
+// RUN: cp %S/* .
+// RUN: dpct -p=%S --out-root=%T --cuda-include-path="%cuda-path/include"
+// RUN: ls Output > default.log
+// RUN: FileCheck --input-file default.log --match-full-lines %T/main.dp.cpp -check-prefix=DEFAULT
+// DEFAULT: main.dp.cpp
```

Comment:
FileCheck comments can be removed, but doesn't block merge

Reply:
fixed